data.sql

```diff
@@ -17,4 +17,34 @@ VALUES ('Charmander', '2020-02-08', 0, false, -11),
 	   ('Angemon', '2005-06-12', 1, true, -45),
 	   ('Boarmon', '2005-06-07', 7, true, 20.4),
 	   ('Blossom', '1998-10-13', 3, true, 17),
-	   ('Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+	   ('Ditto', '2022-05-14', 4, true, 22);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34),
+		('Jennifer Orwell', 19),
+		('Bob', 45),
+		('Melody Pond', 77),
+		('Dean Winchester', 14),
+		('Jodie Whittaker', 38);
+
+INSERT INTO species (name)
+VALUES ('Pokeman'),
+	   ('Digimon');
+
+UPDATE animals
+SET species_id = 1
+WHERE species_id IS NULL;
+
+UPDATE animals
+SET species_id = 2
+WHERE name Like '%mon';
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
+UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
+UPDATE animals SET owner_id = 4 WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
+
+UPDATE species 
+SET name = 'Pokemon'
+WHERE id = 1;
```

Comment:
- [ ] Kudos 👍 for updating the data as suggested earlier, but kindly please update this particular 👇  screenshot with the appropriate query result in accordance to the requirement as well. 

<img width="264" alt="Untitled" src="https://github.com/Ayokunnumi1/vet-clinic/assets/103386183/48613336-315d-4f1d-93b3-7c29af984bbd">